src/applications/disability-benefits/all-claims/components/IntroductionPage.jsx

```diff
@@ -31,7 +31,7 @@ class IntroductionPage extends React.Component {
       return (
         <div className="schemaform-intro">
           <FormTitle title="File for disability compensation" />
-          <FileOriginalClaimPage.component props={this.props} />
+          <FileOriginalClaimPage.Component props={this.props} />
```

Comment:
I suspect that this might be `undefined` - we need to make this match with what is being exported for `FileOriginalClaimPage`

https://github.com/department-of-veterans-affairs/vets-website/blob/cb05808b894496502fe84f9e2b7c85b64adaa8ad/src/applications/disability-benefits/all-claims/components/IntroductionPage.jsx#L13

https://github.com/department-of-veterans-affairs/vets-website/blob/cb05808b894496502fe84f9e2b7c85b64adaa8ad/src/applications/disability-benefits/wizard/pages/file-original-claim.jsx#L29-L32

Here is a [failing test](https://app.circleci.com/pipelines/github/department-of-veterans-affairs/vets-website/5260/workflows/85a1b68f-5d19-490a-981f-6fc43103e535/jobs/31303/tests).

Reply:
correct!